apps/blaze-dashboard/src/pages/setup/components/blaze-private-site.jsx

```diff
@@ -0,0 +1,68 @@
+import { localizeUrl } from '@automattic/i18n-utils';
+import { useTranslate } from 'i18n-calypso';
+import privateSiteGraphic from 'calypso/assets/images/blaze/site-private-graphic@3x.png';
+import { useSelector } from 'calypso/state';
+import { getSelectedSite } from 'calypso/state/ui/selectors';
+
+export default function BlazePrivateSite() {
+	const selectedSiteData = useSelector( getSelectedSite );
+	const siteSlug = selectedSiteData?.slug;
+
+	const translate = useTranslate();
+
+	return (
+		<>
+			<div className="promote-post-i2__inner-container">
+				<div className="promote-post-i2__setup-icon">
+					<img src={ privateSiteGraphic } alt="privete site graphic" />
```

Comment:
I don't think the alt text here is necessary because it's purely for illustrative purposes, but if it is, probably should be translatable and the typo fixed. 


```suggestion
					<img src={ privateSiteGraphic } alt="" />
```